components/layout_2020/flow/mod.rs

```diff
@@ -196,7 +196,7 @@ impl BlockLevelBox {
 
 struct FlowLayout {
     pub fragments: Vec<Fragment>,
-    pub content_block_size: Length,
+    pub content_block_size: Au,
```

Comment:
This change seems independent from "change ResolvedMargins to use Au instead of length".
I think it would be simpler to change the margins alone.
You could split this into 2 PRs, or otherwise at least mention this in the commit description.

Reply:
I apologise for mixing it with margins. I would stick to changing the margins in this PR.